bin/command.php

```diff
@@ -18,25 +19,35 @@
 class Command {
 
 	/**
-	 * Regenerate all doc pages
+	 * Regenerates all doc pages.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--verbose]
+	 * : If set will list command pages as they are generated.
 	 *
 	 * @subcommand gen-all
 	 */
-	public function gen_all() {
+	public function gen_all( $args, $assoc_args ) {
+		// Warn if not invoked with null WP_CLI_CONFIG_PATH.
+		if ( '/dev/null' !== getenv( 'WP_CLI_CONFIG_PATH' ) ) {
```

Comment:
If this is a requirement, why not just `putenv()` it to force it?

Reply:
I did actually consider that, but these are used during bootstrapping so there's no hook available that early? Plus there's an advantage I think to just being explicit and dumb so it's obvious what is happening for such a seldom used command....